src/app/modules/shared_modules/keycard_popup/module.nim

```diff
@@ -24,6 +24,8 @@ export io_interface
 logScope:
   topics = "keycard-popup-module"
 
+const DEFAULT_COLORID_FOR_UNDEFINED_WALLET_ACCOUNT = "undefined"
```

Comment:
Won't be needed to define another constant for the same purpose here if it is defined in `keypair_account_item.nim`.